project-fortis-pipeline/docs/development-library.md

```diff
@@ -0,0 +1,154 @@
+# Commonly Used Libraries
+
+Here is a list of the dependencies we're using on the project, with a brief explanation of their purpose and link to documentation for ease of use.

```

Comment:
This list is very extensive and some of the libraries are very minorly used... you're the expert in this area, but is it really worthwhile to document every little dependency that we have? Isn't this information that someone can just look up from the npm page if they're interested?

Why not instead explain the top level concerns instead of the low level details? E.g. what each of the sub-folders does and the main technologies that it uses?